docs/malformed_files.md

```diff
@@ -0,0 +1,72 @@
+# Dealing with malformed files
+
+## Field compatibility
```

Comment:
Please add a note about running the preprocessor tool here. It will give a summary of malformed/incompatible records.

Reply:
Done